canonical-web-and-design/vanilla-framework
# Improvements to application layout panel responsiveness

## Done

Improves responsive styling of application layout aside panel, provides built in width options and better documentation:
- makes sure aside panel gets unpinned on small screens
- makes sure aside panel doesn't exceed small screen width
- provides 3 width options for panels (is-narrow, is-medium, is-wide)
- improves documentation about application layout panels and customizing them

Fixes #3283

## QA

- Run `./run` or [demo](https://vanilla-framework-3397.demos.haus/docs/examples/)
- Check updated app layout documentation: https://vanilla-framework-3397.demos.haus/docs/layouts/application
- Check application layout panels example https://vanilla-framework-3397.demos.haus/docs/examples/layouts/application/structure
  - make sure aside panel works well responsively (unpins on small screens, is never wider than the screen)
  - check different built-in widths of the panel
- Check other new application layout examples with different panel setups: https://vanilla-framework-3397.demos.haus/docs/examples/
